tests/ssl-opt.sh

```diff
@@ -525,6 +525,14 @@ requires_max_content_len() {
     requires_config_value_at_least "MBEDTLS_SSL_OUT_CONTENT_LEN" $1
 }
 
+CID_MODE=$( get_config_value_or_default "MBEDTLS_SSL_DTLS_CONNECTION_ID_COMPAT" )
+
+requires_cid_compat() {
```

Comment:
Actually I think all tests that use this are duplicated of existing tests that should be removed, and then we can remove this function and the associated variable as well.